Loop/Managers/UserAlert/InAppUserAlertHandler.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  InAppUserAlertHandler.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 4/9/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import LoopKit
+
+public class InAppUserAlertHandler: UserAlertHandler {
+    
+    private weak var rootViewController: UIViewController?
+    
+    private var alertsShowing: [(UIAlertController, UserAlert)] = []
+    private var alertsPending: [(Timer, UserAlert)] = []
+    
+    public init(rootViewController: UIViewController) {
+        self.rootViewController = rootViewController
+    }
+        
+    public func scheduleAlert(_ alert: UserAlert) {
+        switch alert.trigger {
+        case .immediate:
+            show(alert: alert)
+        case .delayed(let interval):
+            schedule(alert: alert, interval: interval, repeats: false)
+        case .repeating(let interval):
+            schedule(alert: alert, interval: interval, repeats: true)
+        }
```

Comment:
I like that idea, but there isn't a current need yet.  The existing system only uses UserNotifications, and `UNTimeIntervalNotificationTrigger` behaves such that it has an initial delay.  At this time I'm only replacing that system...if there's a requirement for it, it would be a separate PR (and JIRA ticket).